test/Sentry.DiagnosticSource.IntegrationTests/SqlListenerTests.verify.cs

```diff
@@ -37,8 +37,13 @@ public async Task RecordsSql()
             var transaction = hub.StartTransaction("my transaction", "my operation");
             hub.ConfigureScope(scope => scope.Transaction = transaction);
             hub.CaptureException(new("my exception"));
-            await TestDbBuilder.AddDataAsync(database);
-            await TestDbBuilder.GetDataAsync(database);
+
+            await using (var connection = await database.OpenNewConnection())
+            {
+                await TestDbBuilder.AddDataAsync(connection);
+                await TestDbBuilder.GetDataAsync(connection);
+            }
```

Comment:
Specifically opening and closing a connection while a Sentry transaction is active